.ansible-lint

```diff
@@ -8,23 +9,22 @@ exclude_paths:
   - .yamllint
   - molecule/
   - tests/azure/
-
+  - meta/runtime.yml
+ 
+kinds:
+  - playbook: '**/tests/**/test_*.yml'
+  - playbook: '**/playbooks/**/*.yml'
 
 parseable: true
 
-quiet: false
+quiet: yes
```

Comment:
What is the result of using quiet here?

Reply:
It actually hides some info that we might want. It should be "false".

Reply:
Fixed.